packet_spec.py

```diff
@@ -31,6 +65,24 @@ class PacketHeader:
 class PacketMessage(ABC):
```

Comment:
We should prob also change this to be a TelemetryPacketMessage since control packets won't have time_since_power_on